app/views/layouts/partials/_form-password-field.blade.php

```diff
@@ -1,5 +1,5 @@
 <div class="field-group {{ setInvalidClass('password', $errors) }}">
   {{ Form::label('password', 'Password: ') }}
-  {{ Form::password('password') }}
+  {{ Form::password('password', array('placeholder' => 'Password must be at least six characters long')) }}
```

Comment:
Looks good, only change is maybe use the literal syntax for the array, so `[ ... ]` instead of `array( ... )`... modernize!